circle.yml

```diff
@@ -6,13 +6,7 @@ test:
     - NODE_ENV=test make client/config/index.js:
         parallel: true
   override:
-    - bin/run-lint :
-        parallel: true
-        files:
-          - client/**/*.js
-          - client/**/*.jsx
-          - server/**/*.js
-          - server/**/*.jsx
+    - bin/run-lint $(git diff --name-only $(git merge-base $(git rev-parse --abbrev-ref HEAD) origin/master)..HEAD *js *jsx)
```

Comment:
> I am sure you can find a way

Would give it a second thought.

 One approach I have already thought was: why not show the warnings too in the CircleCI console but only make CircleCI to fail (by the exit code of the script) if there are any errors in the files modified.

Reason I didn't pursue that way: current run-lint script does not show the warnings, so I took that as an indicator that people look for the green/red state given by CircleCI and only goes to its console if there are some problems.